tools/integration_tests/managed_folders/test_helper.go

```diff
@@ -76,7 +76,7 @@ func providePermissionToManagedFolder(bucket, managedFolderPath, serviceAccount,
 		t.Fatalf(fmt.Sprintf("Error in marshal the data into JSON format: %v", err))
 	}
 
-	localIAMPolicyFilePath := path.Join(os.Getenv("HOME"), "iam_policy.json")
+	localIAMPolicyFilePath := path.Join(os.TempDir(), "iam_policy-"+setup.GenerateRandomString(5)+".json")
```

Comment:
Please use the golang-alternative, which is much cleanear and better-managed.

```suggestion
	localIAMPolicyFilePath := os.CreateTemp(os.TempDir(), "iam_policy-*.json")
```

Also, are we deleting this file somewhere ? If not, we should, for cleanup.

Reply:
Ref https://pkg.go.dev/os#CreateTemp for more details.